include/spdlog/details/line_logger_impl.h

```diff
@@ -46,6 +46,11 @@ inline spdlog::details::line_logger::~line_logger()
 #endif
         _callback_logger->_log_msg(_log_msg);
     }
+
+    if (_log_msg.level >= _callback_logger->_flush_level)
```

Comment:
please use load the value using load(std::memory_order_relaxed) and only then compare. will be faster